src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -413,6 +413,17 @@ def configure_hooks(
         if hasattr(cfg, "algo_backend"):
             self._update_caching_modules(cfg)
 
+        # Update adaptive repeat
+        if not self.training:
+            remove_from_configs_by_type(cfg.custom_hooks, "AdaptiveRepeatDataHook")
+        for custom_hook in cfg.custom_hooks:
```

Comment:
```suggestion
        if not self.training:
            remove_from_configs_by_type(cfg.custom_hooks, "AdaptiveRepeatDataHook")
            return
        for custom_hook in cfg.custom_hooks:
```
How about exit the functon if it isn't training?